### PR TITLE
Forward application logs to fluentd

### PR DESCRIPTION
**Pull request by nabeta**
This PR will fix https://github.com/antleaf/nims-mdr-development/issues/373 .

### CODE REVIEW

File: docker-compose-production.yml [hunk 1]
```diff
@@ -4,6 +4,11 @@
 version: '3'
 
 services:
+  app: &app
+    build:
+      args:
+        FLUENTD_URL: ${FLUENTD_URL}
```
Comment (orangewolf) [inline docker-compose-production.yml:10]: its not clear to me that this build arg gets used anywhere in this pr.  is it needed?

File: fluentd/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+FROM fluent/fluentd:v1.6-debian-1
```
Comment (orangewolf) [inline fluentd/Dockerfile:1]: is this container being used? if so does it need to be in the docker-compose-production file?

File: docker-compose-production.yml (hunk 1 above)
Comment (nabeta) [reply docker-compose-production.yml]: Initially FLUENTD_URL was in a config file and the build failed without the env.
https://github.com/actindi/act-fluent-logger-rails#create-configfluent-loggeryml

I moved the configuration to ``.env`` file and removed this entry from the yaml file.

File: fluentd/Dockerfile (hunk 2 above)
Comment (nabeta) [reply fluentd/Dockerfile:1]: I've added fluentd container to docker-compose-production.
https://github.com/antleaf/nims-hyrax/pull/300/files#diff-2013fcaf2d9b94438426326955a9d1b922017456817d18fc5888bb2b7534f43b

**Review by Steven-Eardley — APPROVED** [review]

File: fluentd/Dockerfile (hunk 2 above)
Comment (Steven-Eardley) [reply fluentd/Dockerfile:1]: The debian image is pretty hefty, (it installs LibreOffice!) but it seems to be working as intended.

**Pull request merged**